libr/bin/bin.c

```diff
@@ -31,6 +31,13 @@ R_LIB_VERSION (r_bin);
 
 #define ARCHS_KEY "archs"
 
+#if !defined(R_BIN_STATIC_PLUGINS)
```

Comment:
it would be better to fix that in the configure-plugins script rather than handling the null case in there, but it's an interesting approach to cover too